2023-02-22 06:40:34,223 [MainThread  ] [INFO ] [root]  STARTING BROWSER: /home/jake/research/chromium/src/out/direct_handles_release/chrome
2023-02-22 06:40:34,223 [MainThread  ] [INFO ] [root]  STARTING BROWSER: driver: /home/jake/research/chromium/src/out/direct_handles_release/chromedriver
2023-02-22 06:40:34,223 [MainThread  ] [INFO ] [root]  STARTING BROWSER: args: --no-default-browser-check --disable-component-update --disable-sync --no-experiments --enable-benchmarking --disable-extensions --no-first-run --disable-background-timer-throttling --disable-renderer-backgrounding --window-size=1500,1000 --enable-logging --log-file=/home/jake/research/v8_handles_experiments/crossbench/chrome_v112_chrome-direct-gc_default/speedometer_2.1/10/browser.chrome.log
2023-02-22 06:40:37,526 [MainThread  ] [INFO ] [root]  RUNNING STORY
2023-02-22 06:42:26,878 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaacec760>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/6486a8969f1b62f0d056364d436f6c24
2023-02-22 06:42:26,879 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaacecc10>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/6486a8969f1b62f0d056364d436f6c24
2023-02-22 06:42:26,879 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fbcaacec0a0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/6486a8969f1b62f0d056364d436f6c24
